packages/ui-components/src/components/form-label/form-label.tsx

```diff
@@ -19,7 +19,7 @@ export class KvFormLabel {
 		return (
 			<Host>
 				<div class="form-field-container">
-					{this.label && (
+					{this.label && this.label !== ' ' && (
```

Comment:
why are you checking this condition here again? 🤔

Reply:
Because if we set the displayLabel to empty string to not display the label the RJSF lib will auto generate a label.